lib/ansible/modules/blockinfile.py

```diff
@@ -288,10 +298,10 @@ def main():
 
     insertbefore = params['insertbefore']
     insertafter = params['insertafter']
-    block = to_bytes(params['block'])
-    marker = to_bytes(params['marker'])
+    block = params['block']
+    marker = params['marker']
     present = params['state'] == 'present'
-    blank_line = [b(os.linesep)]
+    blank_line = [(os.linesep)]
```

Comment:
No need for the parens here, this might cause confusion later that there was an attempt to make this a list of tuples.


```suggestion
    blank_line = [os.linesep]
```